setup.py

```diff
@@ -12,14 +14,35 @@
     version = 'dev'
 
 
-if __name__ == '__main__':
-    setup(name='conda-smithy',
-          version='0.1.0dev0',
-          description='A package to create repositories for conda recipes, and automate their building with CI tools on Linux, OSX and Windows.',
-          author='Phil Elson',
-          author_email='pelson.pub@gmail.com',
-          url='https://github.com/conda-forge/conda-smithy',
-          scripts=[os.path.join('scripts', 'conda-smithy')],
-          packages=['conda_smithy'],
-         )
+def main():
+    skw = dict(
+        name='conda-smithy',
+        version='0.1.0dev0',
+        description='A package to create repositories for conda recipes, and automate '
+                    'their building with CI tools on Linux, OSX and Windows.',
+        author='Phil Elson',
+        author_email='pelson.pub@gmail.com',
+        url='https://github.com/conda-forge/conda-smithy',
+        scripts=[os.path.join('scripts', 'conda-smithy')],
```

Comment:
I think it is better to use an entry point -- without it the script won't be executable on windows. Ran into this with obvious-ci too, see https://pythonhosted.org/setuptools/setuptools.html#automatic-script-creation

Let me know if you want me to do a PR on yours to do it, or we can merge this and I can do a separate PR after this is merged.